lib/fb-user-datastore-client.unit.spec.js

```diff
@@ -137,197 +142,92 @@ test('When encrypting the user’s data', async t => {
 
 // Fetching user data
 test('When requesting user data that exists with a valid key', async t => {
-  t.plan(3)
+  // t.plan(3)
```

Comment:
Reminder

Reply:
Fixed - and removed all the other t.plans